unicode/eval_engine/services.py

```diff
@@ -0,0 +1,48 @@
+import requests
+from editor.models import UserSubmission
+from problems.models import Problem
+from problems.models import ProblemTestCase
+
+# Judge0 assigns an integer to each supported language
+lang = {
+    'C': 4,
+    'C++': 10,
+    'C#': 16,
+    'Clojure': 18,
+    'Java': 26,
+    'JavaScript': 29,
+    'Octave': 32,

```

Comment:
Never head of Octave.  Probably want to remove this, as it most likely won't get support on the client side.

Reply:
Octave is matlab